DPRHTML/features/bottom-pane/main-pane.html

```diff
@@ -50,9 +50,9 @@
       </div>
     </div>
     <div id="main-bottom-pane-tab-panes-Cv" class="main-bottom-pane-tab-pane" data-bind="visible: isCvTabSelected">
-      <div class="cv-container force-show-scrollbar-on-macos">
-        <textarea id="cinput" onkeydown="if(event.keyCode==13) convert(true); event.keyCode=0;"></textarea>
-        <div id="cinform-div">
+      <div class="bottom-container force-show-scrollbar-on-macos">
```

Comment:
Any particular reason why the cc tab code needs to change? I moved it away from bootstrap to css grid to make it more consistent.

Assuming that's ok with you I'll do the same for the Tr tab once you are done. Or if you are familiar with css grid you can also do it.

Reply:
I am not familiar with css grid, please go ahead.